daq/host.py

```diff
@@ -470,8 +522,12 @@ def _monitor_complete(self):
 
     def _monitor_continue(self):
         self._state_transition(_STATE.NEXT, _STATE.MONITOR)
+        self._execute_deferrable_check_actions()
```

Comment:
i'm still horribly confused by the use of the word "deferrable" here... I'm not sure that's the right word. If you could describe what it's doing in a sentence maybe I could suggest an alternative.